### PR TITLE
add a footer template to be easily overridden

### PR DESCRIPTION
**Pull request by voorloopnul**
@ES-Alexander , what if instead of variables we get a template include that we can easily override when using the theme? It feels a bit more flexible for something like a footer

### CODE REVIEW

File: templates/attribution.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<hr class="h-px my-8 bg-gray-200 border-0 dark:bg-gray-700">
+Powered by <a href="" class="text-blue-br">Zola</a>
```
Comment (ES-Alexander) [inline templates/attribution.html:2]:
```suggestion
Powered by <a href="https://www.getzola.org/" class="text-blue-br">Zola</a>
```

**Review by ES-Alexander — COMMENTED** [review]
Hmm, I like the idea of a template so it can be fully replaced, but does it potentially _also_ make sense to include a couple of variables that can optionally be filled in the default template (e.g. the `sponsor` and `code_license` in this case)?

That way simple values can be set via the config, without needing to duplicate the existing file, but more complex ones can just replace the file entirely.

**Comment by voorloopnul** [comment]
> Hmm, I like the idea of a template so it can be fully replaced, but does it potentially _also_ make sense to include a couple of variables that can optionally be filled in the default template (e.g. the `sponsor` and `code_license` in this case)?
> 
> That way simple values can be set via the config, without needing to duplicate the existing file, but more complex ones can just replace the file entirely.

Sounds fair, I'll do that.

**Comment by voorloopnul** [comment]
image
image

image

File: templates/attribution.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<div class="text-gray-500">
+<hr class="h-px my-8 bg-gray-200 border-0 dark:bg-gray-700">
+Powered by
+<a href="https://www.getzola.org/" class="text-blue-br">Zola</a> end
```
Comment (ES-Alexander) [inline templates/attribution.html:4]:
```suggestion
<a href="https://www.getzola.org/" class="text-blue-br">Zola</a> and
```

**Review by ES-Alexander — CHANGES_REQUESTED** [review]
Looking great!

Minor typo - feel free to merge after :-)

**Pull request merged**